src/services/rfqm_service.ts

```diff
@@ -120,6 +120,7 @@ const RFQM_SIGNED_QUOTE_EXPIRY_TOO_SOON = new Counter({
     name: 'rfqm_signed_quote_expiry_too_soon',
     help: 'A signed quote was not queued because it would expire too soon',
 });
+const PRICE_DECIMAL_PLACES = 6;
```

Comment:
I think there should already be this constant in the codebase?

Reply:
Looked around asset swapper (https://github.com/0xProject/protocol/search?p=3&q=decimals) and it's all over the place. Gonna press with this.